lunatrace/bsl/frontend/src/hooks/createLogoutHandler.tsx

```diff
@@ -38,7 +38,10 @@ export function createLogoutHandler(navigate: NavigateFunction) {
   return async () => {
     const resolvedLogoutToken = await logoutToken;
     if (resolvedLogoutToken) {
-      void ory.submitSelfServiceLogoutFlow(resolvedLogoutToken).then(() => navigate('/account/login'));
+      void ory.submitSelfServiceLogoutFlow(resolvedLogoutToken).then(() => {
```

Comment:
Why `void` this here when you already have an `await` up above for `logoutToken`? If this were to fail for some reason, this currently would be silent. It would be nice to catch this (and maybe do something like make a dialogue box?)

Reply:
word, not my code but i will fix, good catch!